### PR TITLE
#38: Changed namespace `exasolvs` to `exasol.evsl`

### LINKED ISSUE #38: Use EVSCL
## Situation

To unify the code base of EVSL and RLSL we need to move common parts to a library called `exasol-virtual-schema-common-lua` (EVSCL).

```[tasklist]
### Tasks
- [x] Include EVSCL
- [x] Update UML Diagrams
- [x] Fix requirement chain
```

```[tasklist]
## Dependencies
- [ ] https://github.com/exasol/exasol-virtual-schema-common-lua/issues/1
- [ ] https://github.com/exasol/exasol-virtual-schema-common-lua/issues/3
```

### PR DESCRIPTION
**Pull request by redcatbear**
Closes #38.

### CODE REVIEW

File: doc/changes/changes_0.5.2.md [hunk 1]
```diff
@@ -6,9 +6,12 @@ Code name: Fix CVE-2023-42503 in test dependency
 
 This release we replaced a testing dependency that was vulnerable to CVE-2023-42503. Production code was not affected.
```
Comment (kaklakariada) [inline doc/changes/changes_0.5.2.md:7]:
```suggestion
In this release we replaced a testing dependency that was vulnerable to CVE-2023-42503. Production code was not affected.
```

File: doc/changes/changes_0.5.2.md [hunk 2]
```diff
@@ -6,9 +6,12 @@ Code name: Fix CVE-2023-42503 in test dependency
...
 
+We also changed the namespace for the virtual schema adapter in the code from `exasolvs` to `exasol.evsl` for uniformity across projects. This has not impact on the function of the virtual schema.
```
Comment (kaklakariada) [inline doc/changes/changes_0.5.2.md:9]:
```suggestion
We also changed the namespace for the virtual schema adapter in the code from `exasolvs` to `exasol.evsl` for uniformity across projects. This has no impact on the function of the virtual schema.
```

File: doc/changes/changes_0.5.2.md [hunk 3]
```diff
@@ -6,9 +6,12 @@ Code name: Fix CVE-2023-42503 in test dependency
...
+
 ## Features
 
-* 45: Fix CVE-2023-42503 in test dependency
+* #38: Changed namespace `exasolvs` to `exasol.evsl`
+* #45: Fix CVE-2023-42503 in test dependency
```
Comment (kaklakariada) [inline doc/changes/changes_0.5.2.md:14]:
```suggestion
* #45: Fixed CVE-2023-42503 in test dependency
```

**Review by kaklakariada — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-build.yml
[the remark this answers is not in the document]
Comment (redcatbear) [reply .github/workflows/ci-build.yml:1]: Updated.

**Review by kaklakariada — DISMISSED** [review]

**Review by kaklakariada — APPROVED** [review]

**Pull request merged**